pkgs/by-name/cl/clevis/package.nix

```diff
@@ -60,6 +60,8 @@ stdenv.mkDerivation (finalAttrs: {
     "man"
   ];
 
+  strictDeps = false; # TODO investigate why
```

Comment:
Perhaps you can expand a bit more why you needed this? Like e.g what happened when you set this to true, or don't set it at all.

Reply:
I need this because I will investigate why strictDeps is breaking the build.
Further, I need to study this in a more in-depth manner, because it is not obvious why some dependencies should  be in, say, `depsBuildBuild`...